### ISSUE TITLE
WCAG 2.0 AA Compliance

### ISSUE THREAD

**Issue by sotirisf** [tkorakas/bg-red#3]
Color blind people may not be able to enjoy the benefits of this great framework, let's find a way to let them know of the red background color.

**Comment by matrixlord** [comment]
WCAG 2.0 AA Compliance should be our top priority even if we have to rethink the framework all over again.

I propose something like,
`*::before {
  content: 'Red'
}`

We have to very careful since millions of websites are using the framework.
